### ISSUE TITLE
ODD not valid...?

### ISSUE THREAD

**Issue by bansp** [laurentromary/stdfSpec#11]
Do I get it correctly that the tei_odds got customized before tei_odds.rnc got derived from it?

For me, the ODD fails validation on the `<persName>` element in the header. I tried with the current tei_odds (3.1.0) and one that was over a year old (version 2.8.0). The tei_odds.rnc provided here with the ODD is 2.9.1.

One way or another, I guess an upgrade of tei_odds is in order, here. I can push the new one, but it will break validation, and I'm not sure which way you guys want to take this issue. I'd suggest conformance with the vanilla tei_odds.

**Comment by laurentromary** [comment]
Can you be more precise and mention the files you are referring to? The are three ODD specifications: standoff-proposal.xml, my current reference one. standoff-proposal_doc.xml : the one which is needed to have a URL from one (me :-}) can chain ODDs. And standoff-proposalPODD.xml the pure ODD version that Lou put together and I need to compare with the initial one, and check chaining, before I make a complete switch.

**Comment by bansp** [comment]
I meant standoff-proposal.xml. I didn't fully understand the purpose of the doc, so I ignored it. 
I'm interested in chaining from this ODD, and was in the process of investigating what gives when I noticed the validation failure.

**Comment by laurentromary** [comment]
the doc is exactly the one you need to chain. Don't touch the files for the time being since I use them for a touchy activity currently. To chain, try:
`<schemaSpec ident="..." prefix="tei_" start="TEI teiCorpus" source="https://raw.githubusercontent.com/laurentromary/stdfSpec/AnnArbor/Specification/standoff-proposal_doc.xml">`

**Comment by bansp** [comment]
Yup, that's what I tried to do with the simpler one, and was wondering if I could pull it off across the references that it contains. Now I see...
But the change in the pull request is of a different nature: these odds don't validate against the current tei_odds, so they break the user-developer contract and users will in effect not use the local copy only because of the baroqueishness of persName, as it if mattered in this case... :-)
Anyway, thanks for the tip!
